src/channel.cpp

```diff
@@ -356,7 +356,8 @@ namespace mamba
                                                      const std::string& path)
     {
         std::string spath = std::string(rstrip(path, "/"));
-        std::string url = URLHandler().set_host(host).set_port(port).set_path(spath).url();
+        std::cout << "spath = " << spath << std::endl;
```

Comment:
happy to merge but this looks like a debug cout?

Reply:
ahem, let' sremove it ;)